roles/collect/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+- name: Collect these facts
+  run_cmds: name=whatever fact_names={{facts_to_collect}}
```

Comment:
Feels like we could have a better *name* attribute than "whatever", I know this is what was originally written. Just a note for improvement.